### PR TITLE
Updated Ragel recipe

### PR DESCRIPTION
**Pull request by miqlas**
For the emperor!

### CODE REVIEW

File: dev-util/ragel/ragel-6.9.recipe [hunk 1]
```diff
@@ -29,6 +29,7 @@ REQUIRES="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	cmd:awk
```
Comment (fbrosson) [inline dev-util/ragel/ragel-6.9.recipe]: If awk is needed, then **`cmd:awk`** should be added to BUILD_**PRE**REQUIRES, not BUILD_REQUIRES.

File: dev-util/ragel/ragel-6.9.recipe [hunk 2]
```diff
@@ -12,9 +12,9 @@ COPYRIGHT="2001-2009 Adrian Thurston
 	2011 Josef Goettgens"
 LICENSE="GNU GPL v1"
 REVISION="1"
-SOURCE_URI="http://www.complang.org/ragel/ragel-6.8.tar.gz"
-CHECKSUM_SHA256="dd7f7d22f3a58147379bda61334d341c2caa0caf9f71897e3e4ec05c8f398764"
-PATCHES="ragel-6.8.patch"
+SOURCE_URI="http://www.colm.net/files/ragel/ragel-6.9.tar.gz"
+CHECKSUM_SHA256="6e07be0fab5ca1d9c2d9e177718a018fc666141f594a5d6e7025658620cf660a"
+PATCHES="ragel-6.9.patch"
```
Comment (miqlas) [inline dev-util/ragel/ragel-6.9.recipe]: Fixed

File: dev-util/ragel/ragel-6.9.recipe (hunk 1 above)
Comment (miqlas) [inline dev-util/ragel/ragel-6.9.recipe]: Fixed.

File: dev-util/ragel/ragel-6.9.recipe [hunk 3]
```diff
@@ -6,15 +6,15 @@ expression machines do, but can also execute code at arbitrary points in the \
 recognition of a regular language.
 Code embedding is done using inline operators that do not disrupt the regular \
 language syntax."
-HOMEPAGE="http://www.complang.org/ragel"
+HOMEPAGE="https://www.colm.net/open-source/ragel/"
 COPYRIGHT="2001-2009 Adrian Thurston
 	2007 Victor Hugo Borja
 	2011 Josef Goettgens"
 LICENSE="GNU GPL v1"
```
Comment (korli) [inline dev-util/ragel/ragel-6.9.recipe]: License is GNU GPL v2.

**Comment by miqlas** [comment]
Fixed.

**Pull request merged**